.gitignore

```diff
@@ -2,3 +2,5 @@
 node_modules
```

Comment:
It should work with or without the `/`. I can add it to node_modules to be consistent.

The build directory is where the static assets get created, I'm assuming it's a create-react-app thing, I was assuming we wouldn't check them into source control, but let me know.